packages/schemas/src/lib/auth/auth-schemas.ts

```diff
@@ -1,19 +1,27 @@
-import { AUTH_METHOD_TYPE_VALUES } from '@lit-protocol/constants';
+import {
+  AUTH_METHOD_TYPE,
+  AUTH_METHOD_TYPE_VALUES,
+} from '@lit-protocol/constants';
 import { z } from 'zod';
 import {
   AuthMethodSchema,
   HexPrefixedSchema,
   NodeSetSchema,
   SessionKeyUriSchema,
 } from '../schemas';
+import { ScopeSchemaRaw } from './ScopeSchema';
+
+export const CustomAuthDataSchema = z.object({
+  authMethodId: HexPrefixedSchema,
+  // This will be a very big number, unlike our native auth
+  authMethodType: z.bigint(),
```

Comment:
Can this value be bigger than Number supports (e.g. > 2^53 ?) If so, we can't represent this data as number (or coerce it to be one like we are trying to do on L24 below)